ijroi/ijroi.py

```diff
@@ -71,10 +74,7 @@ def getfloat():
     roi_type = get8()
     # Discard second Byte:
     get8()
-
-    if roi_type not in [RoiType.FREEHAND, RoiType.POLYGON, RoiType.RECT, RoiType.POINT]:
-        raise NotImplementedError('roireader: ROI type %s not supported' % roi_type)
-
```

Comment:
I agree this looks safe; the interpretation of the header doesn't change depending on the ROI type